assets/eip-6358/src/package-lock.json

```diff
@@ -0,0 +1,10354 @@
+{
```

Comment:
Please remove this file. We generally don't allow build system artifacts in the `assets` directory.